### PR TITLE
[interp] CI some tests in mono/tests 

### PR DESCRIPTION
**Pull request by lewurm**

### CODE REVIEW

**Comment by lewurm** [comment]
@monojenkins build interpreter

**Comment by lewurm** [comment]
@monojenkins build interpreter

**Comment by lewurm** [comment]
@monojenkins build interpreter

**Comment by lewurm** [comment]
hey @akoeplinger and @luhenry, what do you think about this approach to CI mono/tests stuff with the interpreter?  I adopted the `test-runner.cs` a bit so that runtime flags can be passed to the harness. The reason I don't wanna use `MONO_ENV_OPTIONS` is that the `test-runner` also picks up this option (that might be something we don't wanna for other scenarios as well, e.g. `sgen` tests).

File: mono/tests/test-runner.cs [hunk 1]
```diff
@@ -96,6 +97,13 @@ class TestInfo {
 					}
 					runtime = args [i + 1];
```
Comment (akoeplinger) [inline mono/tests/test-runner.cs]: while reviewing I noticed the `--runtime` option is duplicated below, can you please remove the other one?

**Comment by lewurm** [comment]
@monojenkins build interpreter

**Comment by lewurm** [comment]
thanks @akoeplinger, PR updated.

**Comment by lewurm** [comment]
@monojenkins build interpreter

**Pull request merged**